docs/guides/typescript.md

```diff
@@ -428,7 +428,7 @@ function useBearStore<T>(
 }
 ```
 
-You can also make an abstract `createBoundedUseStore` if you create bounded `useStore`s often and want to DRY things up...
+You can also make an abstract `createBoundedUseStore` generator function if you need to create bounded `useStore` hooks often and want to DRY things up...
```

Comment:
A [generator function (`function*`)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*) is a specific concept in JS, so I don't think it's a good name for it.

Reply:
Removed `generator`